examples/basic/poptext/poptext.py

```diff
@@ -0,0 +1,47 @@
+import movis as mv
+import json
```

Comment:
Personally, I prefer that the import order in Python starts with the standard library followed by third-party libraries. Can you modify it like this?

```python
import json

import movis as mv
```